projects/testing/core/textfield.harness.ts

```diff
@@ -1,7 +1,8 @@
 import {TuiComponentHarness} from '@taiga-ui/testing/utils';
 
 export class TuiTextfieldHarness extends TuiComponentHarness {
-    public static hostSelector = 'input[tuiTextfield], textarea[tuiTextfield]';
+    public static hostSelector =
+        'input[tuiTextfieldLegacy], textarea[tuiTextfieldLegacy]';
```

Comment:
Pay attention to these lines during review.
It is okay ?

Reply:
why not?